week-6/Homework/mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,112 @@
 /** Write your CSS in here **/
+body {
+    background-color: orange;
+}
+
+.container {
+    max-width: 80vw;
+    height: 80vh;
+    margin: 10% auto;
```

Comment:
This 10% refers to the `.container` width, which pushes the container upward on narrow screens and downward on wide screens.

How would you fix this?